rep-2000.rst

```diff
@@ -187,10 +190,13 @@ Significant errors discovered in Tier 1 platforms can impact release dates and w
 
 Tier 2
 
-Tier 2 platforms are subject to ad hoc testing and binary packages may not be provided or supported.
+Tier 2 platforms are subject to periodic CI testing which runs both test builds and unit tests with publically accessible results.
```

Comment:
If this publicly accessible results means `ci.ros2.org`, it'd be great to add this precision. Otherwise, it feels like people can setup their own independent CI?

Reply:
> it feels like people can setup their own independent CI?

That is exactly the idea. The CI infrastructure doesn't have to be on ci.ros2.org / build.ros2.org.